parsers/CA_QC.py

```diff
@@ -38,13 +38,13 @@ def if_exists(elem: dict, etype: str):
             return 0.0
 
     data = _fetch_quebec_production()
+    list_res = []
     for elem in reversed(data["details"]):
         if elem["valeurs"]["total"] != 0:
-
-            return {
+            list_res.append(dict({
```

Comment:
`dict({})` is redundant - just `{}` also creates a dict (this is called a "dict literal"). Same on line 47 below

Reply:
Thanks @jarek. I've made changes accordingly.